staging/cse/windows/provisioningscripts/kubeletstart.ps1

```diff
@@ -48,6 +55,29 @@ if ($global:KubeBinariesVersion -lt "1.27.0") {
     $KubeletArgList += @("--container-runtime=remote")
 }
 
+# Run the secure TLS bootstrap script to generate a kubelet client certificate, if enabled
+if ($global:EnableSecureTLSBootstrapping) {
+    $aadResource = $global:DefaultSecureTLSBootstrapAADResource
+    if ($global:CustomSecureTLSBootstrapAADResource) {
+        $aadResource = $global:CustomSecureTLSBootstrapAADResource
+    }
+    & "c:\k\securetlsbootstrap.ps1" -KubeDir $global:KubeDir -APIServerFQDN $global:MasterIP -AADResource $aadResource
```

Comment:
kubeletstart.ps1 will be run when
- Node is provisioned
- Node is restarted
- after kubelet crashed
Is it expected to always call securetlsbootstrap.ps1 before starting kubelet?

Reply:
what will happen if securetlsbootstrap.ps1 found that the cert is expired but it failed in getting new cert?

Reply:
And how will it handle the expired cert when kubelet is running?

Reply:
kubelet is responsible for rotating its own certificate after its bootstrapped with an initial one (via the `--rotate-certificates` flag)

in cases where the bootstrap client sees that the kubeconfig references an expired certificate (or a certificate signed by an out-of-date cluster CA), it will initiate the bootstrap process